docs/scripts/Pipfile

```diff
@@ -8,7 +8,7 @@ verify_ssl = true
 [packages]
 sphinx-autobuild = "*"
 sphinx-copybutton = "==0.2.12"
-sphinx = "==1.8.3"
+sphinx = "==4.3.1"
```

Comment:
I had not realized we were so far behind. 🙈

Reply:
that’s why I got somewhat scared :)